README.asciidoc

```diff
@@ -304,7 +304,75 @@ hook global WinSetOption filetype=<language> %{
 }
 ----
 
-The faces used for semantic tokens and modifiers can be modified in `kak-lsp.toml`, under the `semantic_tokens` and `semantic_token_modifiers` sections. The modifiers are used first if available, and then the main token type is used if no modifier face is specified.
+The faces used for semantic tokens and modifiers can be modified in `kak-lsp.toml`, under the `semantic_tokens` section.
+The syntax for such an entry is
+
+[source,toml]
+----
+[[semantic_tokens]]
+name = "variable"
```

Comment:
Nit: I think the spec calls this "token type". Maybe we should call it "token" or "token_type"?